### PR TITLE
Add ollama documentation

### PR DESCRIPTION
**Pull request by cescoffier**
This commit introduces the initial documentation for Ollama:

- Write the ollama documentation
- Change model defaults to be correct
- Explain how to pull model (which then avoid using a dev service)
- Explain how to use the dev service

### CODE REVIEW

**Review by geoand — APPROVED** [review]
Nice!

File: docs/modules/ROOT/pages/ollama.adoc [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+https://ollama.com/[Ollama] proposes a way to run large language models (LLMs) locally.
+You can run many https://ollama.com/library[models] such as LLama2, Mistral, or CodeLlama on your local machine.
+
+[#_prerequisites]
+== Prerequisites
+
+To use Ollama, you need to have a running Ollama _server_.
+Go to the https://ollama.com/download[Ollama download page] and download the server for your platform.
+
+Once install, check that Ollama is running using:
```
Comment (gsmet) [inline docs/modules/ROOT/pages/ollama.adoc:14]:
```suggestion
Once installed, check that Ollama is running using:
```

File: docs/modules/ROOT/pages/ollama.adoc [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+----
+
+It may not display any model, which is fine, let's pull the `llama2` model:
+
+[source,shell]
+----
+> ollama pull llama2
+----
+
+WARNING: Models are huge. For example llama2 is 3.8Gb. Make sure you have enough disk space.
```
Comment (gsmet) [inline docs/modules/ROOT/pages/ollama.adoc:28]: Not sure about the case of LLama2 but we should try to be consistent.
```suggestion
WARNING: Models are huge. For example LLama2 is 3.8Gb. Make sure you have enough disk space.
```

File: docs/modules/ROOT/pages/ollama.adoc [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+[source,shell]
+----
+> ollama pull nomic-embed-text
+----
+
+[NOTE]
+.Dev Service
+====
+If you have ollama running locally, you do not need a dev service.
```
Comment (gsmet) [inline docs/modules/ROOT/pages/ollama.adoc:40]:
```suggestion
If you have Ollama running locally, you do not need a dev service.
```

**Comment by geoand** [comment]
I think the ollama `-deployment` artifact is missing from docs

**Review by gsmet — COMMENTED** [review]
Always curious to see the progress here so I had a quick look.
Added a couple of suggestions.

**Comment by cescoffier** [comment]
Thanks @gsmet !

**Pull request merged**